app/services/SubmissionService.scala

```diff
@@ -81,6 +81,8 @@ class SubmissionService @Inject()
           } yield futureSeq :+ future
       )
 
+    Logger.info(s"[SubmissionService][submit] Submitting")
```

Comment:
There's no substitute in the logging string

Reply:
Removed interpolation from logging statement.